assignment1 from Arjun Chaudhary

arjun0012 opened the pull request:
I have set the array while declaring it and not taken input at stdin for the array, let me know if I have to do that.

tg-trainer commented:
@arjun0012 please remove all the target folder and add it into the .gitignore folder.

Also with .idea & .bsp

arjun0012 commented:
> @arjun0012 please remove all the target folder and add it into the .gitignore folder.
> 
> Also with .idea & .bsp

I Have removed the extra files and implemented the method discussed in the meeting. Please have a look.
